opencog/atomspace/ImportanceIndex.cc

```diff
@@ -123,3 +146,35 @@ UnorderedHandleSet ImportanceIndex::getHandleSet(
 	               [](Atom* atom)->Handle { return atom->getHandle(); });
 	return ret;
 }
+
+UnorderedHandleSet ImportanceIndex::getMaxBinContents()
+{
+    UnorderedHandleSet ret;
+    for (AtomSet s : boost::adaptors::reverse(idx))
+    {
+        if (s.size() > 0)
+        {
+	        std::transform(s.begin(), s.end(), inserter(ret),
+	               [](Atom* atom)->Handle { return atom->getHandle(); });
+	        return ret;
+        }
+    }
+
+    return ret;
+}
+
+UnorderedHandleSet ImportanceIndex::getMinBinContents()
+{
+    UnorderedHandleSet ret;
+    for (AtomSet s : idx)
```

Comment:
I am guessing that what you really want is to get rid of the for-loop, and just set
```
const AtomSet& s = idx[0];
```
and for the other one, 
```
const AtomSet& s = idx[IMPRTANCE_MAX_BINS-1];
```